test/msw-api/exception-handling.test.ts

```diff
@@ -28,4 +28,8 @@ test('transforms uncaught exceptions into a 500 response', async () => {
       'ReferenceError: nonExisting is not defined',
     ),
   })
+
+  const consoleError = runtime.consoleSpy.get('raw').get('error').join('\n')
```

Comment:
You can use `consoleSpy.get('error')` straight away to retrieve a list of errors. Then, you can write an assertion that checks if a certain phrase occurs in that list:

```ts
const errors = runtime.consoleSpy.get('error')
expect(errors).toEqual(expect.arrayContaining([
  expect.stringContaining('ReferenceError: ...'),
  // By the way, not sure if this is not a brittle assertion
  // as it relies on implementation details of the "RestHandler" class.
  // Perhaps we can rephrase this so it'd be abstract. 
  expect.stringContaining('at RestHandler.run')
]))
```

Reply:
Ah thanks, I'm still getting used to `expect`. Wrt the rest handler, yes that's brittle. I was not 100% sure, but I've changed it to just `"  at "` which is a bit generic, but I think works for this particular case.